### DISCUSSION TITLE
Idea: Library Status module

### DISCUSSION THREAD

**Discussion by hawkrives** [StoDevX/AAO-React-Native#5543] [Ideas]
I'm not looking for a full replacement for the library website; I'd find it really useful to see what books I have checked out, though. 

- ~~Search the catalog~~
- ~~Check out books~~
- ~~Check out bicycles~~
- See what books I have checked out and when they're due

I really only want the last feature, as a first take. 

The others will all take significantly more engineering, I believe.

**Comment by drewvolz** [comment]
Something something long laundry list of notes I have 

Library curl  All Results? curl 'https://stolaf-primo.hosted.exlibrisgroup.com/primo_library/libweb/webservices/rest/primo-explore/v1/pnxs?q=any,contains,bikes&qExclude=&qInclude=&rtaLinks=true&scope=default_scope&sort=rank&tab=library_collections&vid=01BRC_SOC&inst=01BRC_SOC' -H 'X-From-ExL-API-Gateway: 1' --compressed | pbcopy
 Resource Recommender
curl 'https://stolaf-primo.hosted.exlibrisgroup.com/primo_library/libweb/webservices/rest/v1/resourceRecommender?lang=en_US&query=bike&view=01BRC_SOC' -H 'X-From-ExL-API-Gateway: 1' --compressed
 __

blocks_messages
{path: "/primo_library/libweb/webservices/rest/v1/myaccount/blocks_messages", method: "GET"}

cancel_requests
{path: "/primo_library/libweb/webservices/rest/v1/myaccount/cancel_requests", method: "POST"}

collection
{path: "/primo_library/libweb/webservices/rest/v1/myaccount/collection", method: "GET"}

counters
{path: "/primo_library/libweb/webservices/rest/v1/myaccount/counters", method: "GET"}

fines
{path: "/primo_library/libweb/webservices/rest/v1/myaccount/fines", method: "GET"}

loans
{path: "/primo_library/libweb/webservices/rest/v1/myaccount/loans", method: "GET"}

personal_settings
{path: "/primo_library/libweb/webservices/rest/v1/myaccount/personal_settings", method: "GET"}

renew_all_loans
{path: "/primo_library/libweb/webservices/rest/v1/myaccount/renew_all_loans", method: "POST"}

renew_loan
{path: "/primo_library/libweb/webservices/rest/v1/myaccount/renew_loan", method: "POST"}

requests
{path: "/primo_library/libweb/webservices/rest/v1/myaccount/requests", method: "GET"}

update_personal_settings
{path: "/primo_library/libweb/webservices/rest/v1/myaccount/update_personal_settings", method: "POST"}


curl 'https://stolaf-primo.hosted.exlibrisgroup.com/primo_library/libweb/webservices/rest/primo-explore/v1/pnxs/L/' --data-binary '{"inst":"01BRC_SOC"}'  -H 'X-From-ExL-API-Gateway: 1' --compressed

https://stolaf-primo.hosted.exlibrisgroup.com/primo_library/libweb/webservices/rest/primo-explore/v1/pnxs?limit=200&vid=01BRC_SOC

**Comment by rye** [comment]
I like the "currently checked out" as a good first thing to build.
